inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/sinks/PulsarSink.java

```diff
@@ -196,7 +196,6 @@ public void destroy() {
             AgentUtils.silenceSleepInMs(batchFlushInterval);
         }
         shutdown = true;
-        EXECUTOR_SERVICE.shutdown();
```

Comment:
Then, when we shutdown this global thread pool?

Reply:
The `PulsarSink` thread pool is global and, based on the current design, will always exist throughout the agent's lifecycle. When closed, new tasks cannot be submitted to the thread pool; think it's okay not to shutdown it.

Reply:
Ok